### DISCUSSION TITLE
How do I use the command decorator with MyPy strict enabled?

### DISCUSSION THREAD

**Discussion by Owen-OptiGrid** [fastapi/typer#1008] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the Typer documentation, with the integrated search.
- [X] I already searched in Google "How to X in Typer" and didn't find any information.
- [X] I already read and followed all the tutorials in the docs and didn't find an answer.
- [X] I already checked if it is not related to Typer but to [Click](https://github.com/pallets/click).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
import typer
from typer import Typer

app: Typer = typer.Typer()

@app.command()
def main() -> None:
    print("Hello")

if __name__ == "__main__":
    app()
```

### Description

I ran MyPy in strict mode and it complains about the command decorator:

run.py:7: error: Untyped decorator makes function "main" untyped  [misc]

Is there a way or an example of using the command decorator in a way that keeps MyPy happy?

### Operating System

Windows

### Operating System Details

Windows 11 version 23H2

### Typer Version

0.12.5

### Python Version

3.12.6

**Comment by YuriiMotov** [comment, marked as answer]
Just ran this code with Typer 0.17.4 and mypy 1.18.1 and got `Success: no issues found in 1 source file`.

So, I think this has been fixed

**Comment by Owen-OptiGrid** [comment]
I'd forgotten about this question... yeah don't think I'd noticed it lately.
